### ISSUE TITLE
Create a neural network

### ISSUE THREAD

**Issue by alvinadar** [alvinadar/Google-Ads-sales-prediction-using-neural-network#2]
Define the input of the connection
Activation function for hidden and output
Compile using the optimizer  loss function

**Comment by qaisy21** [comment]
Hi Guys, I Have already create a neural network:                                                                                                                                         https://colab.research.google.com/drive/1iQTxQpte-jS1n1K2XpVKPTKnm00lF02q?usp=sharing                                                                 @alvinadar @khankanmariam @jerrywong69
